modules/api/modules/v1/controllers/DefaultController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace app\modules\api\modules\v1\controllers;
+
+use Yii;
+use app\modules\api\modules\v1\models\User;
+use app\modules\api\modules\v1\Module;
+use app\modules\api\modules\v1\components\BaseController;
+use yii\filters\AccessControl;
+use yii\helpers\Url;
+
+class DefaultController extends BaseController
+{
+    /**
+     * @inheritdoc
+     */
+    public function behaviors()
+    {
+        $behaviors = parent::behaviors();
+
+        $behaviors['access'] = [
+            'class' => AccessControl::class,
+            'rules' => [
+                [
+                    'allow' => true,
+                    'actions' => ['index', 'profile'],
+                ],
+            ],
+        ];
+
+        return $behaviors;
+    }
+
+    /**
+     * @return array
+     */
+    public function actionIndex()
+    {
+        return [
+            'version' => Module::VERSION,
+            'urls' => [
+                'docs' => Url::to(['/api/docs'], true),
+                'news' => Url::to(['news/index'], true),
```

Comment:
Should there be `/` at the beginning?

Reply:
Maybe i did not need to insert a link to the documentation on the site. Since other links lead to the API.

Now:
```
{
    "status": "ok",
    "data": {
        "version": 1,
        "urls": {
            "docs": "http://yiifeed.com/api/docs",
            "news": "http://yiifeed.com/api/v1/news",
            "users": "http://yiifeed.com/api/v1/users"
        }
    }
}
```

To make the code like below, we would have to add Model. Thought it was not necessary. It's easier to completely remove this information.
```
{
    "status": "ok",
    "data": {
        "version": 1,
        "docsUrl": "http://yiifeed.com/api/docs",
        "_links": {
            "news": {
                "href": "http://yiifeed.com/api/v1/news"
            },
            "users": {
                "href": "http://yiifeed.com/api/v1/users"
            }
        }
    }
}
```

Reply:
Did second option through array.